packages/headless/src/api/commerce/commerce-api-params.ts

```diff
@@ -28,6 +28,7 @@ export interface ContextParam {
 export interface ContextParams {
   view: ViewParams;
   user?: UserParams;
+  product?: {productId: string};
```

Comment:
We should extract this in a separate interface like is done for other params
```suggestion
  product?: ProductParam;
```